pkg/reconciler/knativeserving/knativeserving_controller.go

```diff
@@ -55,35 +52,26 @@ var (
 // Reconciler implements controller.Reconciler for Knativeserving resources.
 type Reconciler struct {
 	*reconciler.Base
-	// Listers index properties about resources
-	knativeServingLister listers.KnativeServingLister
-	config               mf.Manifest
-	servings             map[string]int64
+	config   mf.Manifest
+	servings map[string]int64
 	// Platform-specific behavior to affect the transform
 	platform common.Platforms
 }
 
 // Check that our Reconciler implements controller.Reconciler
-var _ controller.Reconciler = (*Reconciler)(nil)
+var _ knsreconciler.Interface = (*Reconciler)(nil)
 
 // Reconcile compares the actual state with the desired, and attempts to
 // converge the two. It then updates the Status block of the Knativeserving resource
 // with the current status of the resource.
-func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
+func (r *Reconciler) ReconcileKind(ctx context.Context, original *servingv1alpha1.KnativeServing) pkgreconciler.Event {
 	// Convert the namespace/name string into a distinct namespace and name
-	namespace, name, err := cache.SplitMetaNamespaceKey(key)
+	key, err := cache.MetaNamespaceKeyFunc(original)
 	if err != nil {
 		r.Logger.Errorf("invalid resource key: %s", key)
 		return nil
 	}
-	// Get the KnativeServing resource with this namespace/name.
-	original, err := r.knativeServingLister.KnativeServings(namespace).Get(name)
```

Comment:
I see build failures because of this line, are the tests stale?

Reply:
pkg/reconciler/knativeserving/knativeserving_controller.go:256:21: r.knativeServingLister undefined (type *Reconciler has no field or method knativeServingLister) specifically, which maybe is somewhere else I'm missing in the file :)